nilearn/datasets/atlas.py

```diff
@@ -59,12 +59,18 @@ def fetch_atlas_difumo(dimension=64, resolution_mm=2, data_dir=None, resume=True
 
     Returns
     -------
-    data : sklearn.datasets.base.Bunch
+    data : :func:`sklearn.utils.Bunch`
```

Comment:
```suggestion
    data : :class:`sklearn.utils.Bunch`
```

Or I guess `:obj:` would work.

EDIT: Although, I guess it doesn't really matter if the link works...

Reply:
Using `:class:` doesn't seem to work. After looking at https://github.com/scikit-learn/scikit-learn/blob/0d378913be6d7e485b792ea36e9268be31ed52d0/doc/modules/classes.rst, I used `:func:` instead to make it work. I think it might be a small bug in scikit-learn's doc.

Reply:
Oh... thanks for clarifying. Really glad I didn't comment on every line where it was used!